_posts/2016-03-28-Mutual-Problems.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+name: mutual-problems
+title: Mutual Problems
+date: 2016-03-28 00:00:00 +00:00
+author: Robert Firek
+image:
+    src: /assets/img/custom/blog/2016-03-28-https.jpg
+    attribution:
+        text: Image by Sean MacEntee (CC BY 2.0)
+        href: https://www.flickr.com/photos/smemon/15944989872/in/photostream/
+tags:
+- Java
+- HTTPS
+- Mutual Authentication
+--- 
+
+The HTTPS protocol is the well-established standard for securing our internet connections. Understanding how this protocol works is not problem and the corresponding [RFC document](https://tools.ietf.org/html/rfc2818) is available since 2000.
+
+Despite HTTPS is used so widely, you can still find a software which doesn't handle this protocol without unnecessary complexity. Unfortunately I'v experienced  problems during the implementation of [the mutual authentication](https://en.wikipedia.org/wiki/Mutual_authentication) in the language which should not surprise me at all. It is **Java**.
```

Comment:
Despite the fact that HTTPS is used so widely, you can still find software that doesn't handle this protocol in a simple way. A lot of software modules add unnecessary complexity to their HTTPS implementation.

Several suggested improvements
 * Despite the fact that
 * No "a" in front of software
 * which -> that
 * remove the double negative and moved the "unnecessary complexity" part to another sentence